products/appengine/terraform.yaml

```diff
@@ -12,6 +12,7 @@
 # limitations under the License.
 
 --- !ruby/object:Provider::Terraform::Config
+legacy_name: "appengine"
```

Comment:
I haven't followed all the discussion here, so you may know this already - this resource is unreleased and you may want to standardize on `app_engine` for consistency with Paddy's work on `google_app_engine_application`.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/issues/1210